src/main/java/com/epam/preprod/sirenko/PrintToConsole.java

```diff
@@ -0,0 +1,11 @@
+package com.epam.preprod.sirenko;
+
+public class PrintToConsole {
+	
+	private PrintToConsole() {
```

Comment:
Nice! Its good practice to create private default constructor for classes with util static methods.

Reply:
Yay, it really is a good practice to read sonar's suggestions :)